style.css

```diff
@@ -0,0 +1,135 @@
+/*Color palette*/
+/*light yellow #e9f089*/
+/*#f14e4e - red - 'Game Over'*/
+/*#f1bb4e - Yellow - 'Player'*/
+/*#84f14e - limegreen - ''*/
+/*#4efl8f - green - ''*/
+/*#4e9afl - blue - ''*/
+/*9a4efl - purple - ''*/
+/*#f14ebd - pink - ''*/
+/*#ded9d1 - lt grey - ''*/
+
+body {
+}
+
+div {
+  border-style: 1px solid red;
+}
+
+.container {
+  display: grid;
+  width: 50%;
+  height: auto;
+  margin: auto;
+  background: #ded9d1;
+  justify-content: center;
+  
+}
+
+.title {
+  padding-top: 40px;
+  text-align: center;
+  color: #f14ebd;
+ //text-shadow: 1px 1px 2px;
```

Comment:
Incorrect commenting for css.
<img width="221" alt="image" src="https://github.com/phaynie/tictactoe/assets/16355881/5a907a31-ffd3-4ca2-99c2-d8d7523fec4f">

Should be like this:
```suggestion
 /* text-shadow: 1px 1px 2px; */
```